jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/PayloadParser.java

```diff
@@ -20,18 +20,31 @@
 import java.io.OutputStream;
 import java.nio.ByteOrder;
 import java.util.Arrays;
+import java.util.HashMap;
+
+import org.apache.tomcat.util.buf.HexUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import us.dot.its.jpo.ode.util.CodecUtils;
 
 public class PayloadParser extends LogFileParser {
 
-   public static final int PAYLOAD_LENGTH_LENGTH = 2;
+   private static Logger logger = LoggerFactory.getLogger(PayloadParser.class);
+   private static HashMap<String, String> msgStartFlags = new HashMap<String, String>();
 
+   private static final int HEADER_SIZE = 20; 
+   public static final int PAYLOAD_LENGTH_LENGTH = 2;
```

Comment:
Copy/pasta or do we need the "_length_length" ?

Reply:
Ah I see it was already in here above...any idea why the name?

Reply:
I have no idea, haha! I can change it though.